code/datums/records/manifest.dm

```diff
@@ -160,3 +160,36 @@ GLOBAL_DATUM_INIT(manifest, /datum/manifest, new)
 
 	target.rank = assignment
 	target.trim = trim
+
+/datum/manifest/ui_state(mob/user)
```

Comment:
i noticefd we had a type that didnt need to exist in /datum/crew_manifest when i was working on making them show up on crew manifest so i put that here hope no one minds >w<